lib/helpers/index.js

```diff
@@ -8,6 +10,13 @@ exports.buildQueryRequest = ({ skip, limit, sort, projection, find }) => {
     result.options.skip = skip || 0
     result.options.limit = limit || 100
     if (find) result.query = JSON.parse(find)
+    // URL encoded _id is converted to ObjectId as it will be sent as string
```

Comment:
It feels like we should abstract this logic in a separate helper method to keep this code clean. I can also still see `result?.query?` even though they are set on line 4